packages/server/shared/src/lib/blocks/step-output-transformer.ts

```diff
@@ -0,0 +1,33 @@
+import { FileCompression } from '@openops/shared';
+import { fileCompressor } from '../file-compressor';
+import { encryptUtils } from '../security/encryption';
+
+export const stepOutputTransformer = {
```

Comment:
@MarceloRGonc is it really shared?
isn't the engine only encrypting, and the server only decrypting?